appimagebuilder/tester/execution_test.py

```diff
@@ -74,12 +76,15 @@ def run(self):
         finally:
             container.kill()
 
-    def _run_command(self, command, container, user="root"):
+    def _run_command(self, command, container, user="root", accepted_exit_codes=None):
+        if accepted_exit_codes is None:
+            accepted_exit_codes = [0]
+
```

Comment:
Hmm, using `None` rather than an empty list `[]` as the default argument made me confused, but I found the [doc](https://docs.python.org/3/reference/compound_stmts.html#function-definitions) which said about it. Thanks, I learned something useful from you.

Reply:
I didn't knew about it either but my IDE (PyCharm) suggested it.